src/Project/BuilderCfgFile.ts

```diff
@@ -153,7 +152,7 @@ export class BuilderCfgFile extends EventEmitter implements helpers.FileSelector
 
     console.log('importTFlite = ', importTFlite);
     this.jobOwner.addJob(importTFlite);
-    this.logger.outputLine('Add Import: ' + inputModel);
```

Comment:
IMHO, every `Logger.append(..)` in each function should replaced to `Logger.info(this.tag, ...)` since it cannot guarantee its information has been printed before. 

For example, this line cannot sure that `cfgImportTf` already executed before `cfgImportTflite`.. If I'm misunderstood something, please let me know.. 😺

Reply:
I changed `Logger.append` in `BuildCfgFile.ts` to `Logger.info`.

Log screen seems OK.

![image](https://user-images.githubusercontent.com/63768207/169950042-a86a5fc9-5418-4b36-9fec-3ec9324d69b5.png)